mzLib/Proteomics/Protein/Protein.cs

```diff
@@ -235,32 +219,51 @@ public string GetEnsemblFastaHeader()
 
         /// <summary>
         /// Gets peptides for digestion of a protein
+        /// Legacy
+        /// </summary>
+        public IEnumerable<PeptideWithSetModifications> Digest(DigestionParams digestionParams,
```

Comment:
Obsolete tag needed?

Reply:
I do not think so. If one wanted to digest but only cared about proteins (such as in the XL or Glyco search task) this method would be called.